client/src/pages/Home.tsx

```diff
@@ -0,0 +1,71 @@
+import NavigationBar from "../components/NavigationBar.tsx";
```

Comment:
Since we will use the same NavigationBar for multiple pages, you can include it in the BrowserRouter. Just pass Homepage as a child. We then can add more children to the NavBar, without importing them into individual files. The [same thing](https://github.com/Vladmidir/Your-Movie-List/blob/main/client/src/App/App.js) can be done with the Footer too I think. I remember I did that in my YourMovieList project, you can take a look at that for reference. Or just google it. 

![image](https://github.com/utmgdsc/UML_Mentor/assets/121591697/9d50246c-a998-4b34-befa-7c8583d2c8b7)

Reply:
P.S. You gonna need to add the "Outlet" tag to the NavBar and Footer.

Reply:
Good idea